### ISSUE TITLE
panic on use of cloned `BuiltEntityClone`

### ISSUE THREAD

**Issue by sapir** [Ralith/hecs#253]
The following triggers a panic on current master (9d9951d)

```rs
use hecs::*;

#[derive(Clone)]
struct A;

fn main() {
    let mut world = World::new();

    let mut builder = EntityBuilderClone::new();
    builder.add(A);
    let built_entity = builder.build();

    let built_entity = built_entity.clone();

    world.spawn(&built_entity);
}
```

Changing `impl Clone for Common<DynamicClone>` @ https://github.com/Ralith/hecs/blob/9d9951dd663ff90f1b89d76e465009829565c2bc/src/entity_builder.rs#L406 to clone the `ids` field seems to prevent the panic.

**Comment by Ralith** [comment]
Thanks for the excellent report! Fix at https://github.com/Ralith/hecs/pull/254.

**Comment by Ralith** [comment]
Fix released in 0.7.6.
